File: multi_lang/go_templates/api_template.go
// Template: Gin API skeleton (raw). Placeholders substituted by GoApiGenerator.
package main

import (
	"net/http"

	"github.com/gin-gonic/gin"
)

func healthz(c *gin.Context) {
	c.JSON(http.StatusOK, gin.H{"status": "ok"})
}

func list__MODEL_NAME__(c *gin.Context) {
	c.JSON(http.StatusOK, gin.H{"service": "__MODULE_NAME__", "items": []})
}

func main() {
	r := gin.Default()
	r.GET("/healthz", healthz)
	r.GET("/api/v1/__ROUTE__", list__MODEL_NAME__)
	r.Run(":8080")
}

File: multi_lang/python_templates/api_template.py
"""Template: FastAPI service skeleton (raw). Placeholders are substituted by PythonApiGenerator."""
from fastapi import FastAPI, HTTPException
from pydantic import BaseModel

__MODULE_NAME__app = FastAPI(title="__MODULE_NAME__")


class __MODEL_NAME__Request(BaseModel):
    __FIELDS__


@app.get("/api/v1/__ROUTE__")
def list_items() -> dict:
    """Return a stub listing for the __MODULE_NAME__ resource."""
    return {"service": "__MODULE_NAME__", "items": []}


@app.get("/healthz")
def healthz() -> dict:
    """Liveness probe used by the orchestrator and Kubernetes."""
    return {"status": "ok"}